lib/discover/global_test.go

```diff
@@ -84,7 +85,7 @@ func TestGlobalOverHTTP(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
-	if len(direct) != 1 || direct[0] != "tcp://192.0.2.42::22000" {
+	if len(direct) != 1 || direct[0] != "tcp://192.0.2.42::22000?prio=100" {
```

Comment:
Heh. Double colons are everywhere in the test case. I'm amazed that survives the various url.Parse() that happen along the way, it's obviously incorrect. Lacking the slash is fine though.